stdlib/tkinter/ttk.pyi

```diff
@@ -1071,26 +1083,21 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
     @overload
     def tag_bind(self, tagname: str, *, callback: str) -> None: ...
     @overload
-    def tag_configure(self, tagname: str, option: Literal["text"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["image"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["anchor"]) -> tkinter._Anchor | Literal[""]: ...
-    @overload
```

Comment:
Is there a reason to delete `"text"` and `"anchor"`?

```python
>>> from tkinter.ttk import Treeview
>>> tv = Treeview()
>>> tv.tag_configure(foo, text='lol')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'foo' is not defined
>>> tv.tag_configure('foo', text='lol')
{}
>>> tv.tag_configure('foo')
{'text': 'lol', 'image': '', 'anchor': '', 'background': '', 'foreground': '', 'font': ''}
```

Reply:
`ttk_treeview(3tk)` lists the following under the heading "TAG OPTIONS":

- `-foreground`: Specifies the text foreground color.
- `-background`: Specifies the cell or item background color.
- `-font`: Specifies the font to use when drawing text.
- `-image`: Specifies the item image, in case the item's `-image` option is empty.

The same list appears in [the Python docs](https://docs.python.org/3/library/tkinter.ttk.html#tag-options) and [on TkDocs](https://tkdocs.com/tutorial/tree.html).

I tried setting "text", "anchor", and "font" on a Treeview just now; the font option had the expected effect, but "text" and "anchor" had no effect I could see. I'm not even sure what "text" or "anchor" would logically do for a tagged cell in a Treeview; the cell's text comes from the relevant field of the underlying item, and the anchoring (alignment) can be specified on the column.

I assumed they were copy/pasted from the `heading()` definition.